evm/src/helpersV2.ts

```diff
@@ -278,13 +278,23 @@ export async function fulfillOracleRequest(
     gasLimit: 1000000, // FIXME: incorrect gas estimation
   },
 ): ReturnType<typeof oracleContract.fulfillOracleRequest> {
+  const d = debug.extend('fulfillOracleRequest')
```

Comment:
There should probably be a pointer in `./debug` to the documentation for the `debug` package, along with brief instructions on how to turn on debugging output.

Reply:
Yeah sounds good, ill add this in the future